zipline.gemspec

```diff
@@ -16,6 +16,9 @@ Gem::Specification.new do |gem|
   gem.version       = Zipline::VERSION
   gem.licenses      = ['MIT']
 
+  gem.required_ruby_version = ">= 2.3"
```

Comment:
Ok, I just saw #83 after pushing the change. This means that unless we adapt the code to work with Rubies <2.7 Rails 5.2 is now also unsupported, see https://github.com/rails/rails/issues/38426.

So this should be changed to `>= 2.7`, the `actionpack` dependency to `>= 6.0` and _theoretically_ that means we could use the code that's in Rails itself, even if it is private and not use the gem.

What do you think, @fringd?

Reply:
Thanks for all the work! let's keep it like this using the gem and bump to 2.7 i think?

Reply:
Done. I assume this would warrant a 2.0 release of `zipline`?